resources/kustomization.yaml

```diff
@@ -13,5 +13,4 @@ resources:
   - base/cluster-role-binding.yaml
   - base/crds/minio.min.io_tenants.yaml
   - base/service.yaml
-  - base/deployment.yaml
```

Comment:
we definetively need this in our base resources

Reply:
I put this back

Reply:
this file is still out of `/resources` we need this here to make sure installing via `kubectl apply -k github.com/minio/operator/resources/\?ref\=v4.3.9` still works, you can place the other kustomizations in a parent folder such as `/testing` so that you don't run into problems

Reply:
Good idea, put under `\testing`, it is working on my MacBook Air. Modified PR to test on Virtual Environment of GitHub Actions in Ubuntu. I guess this is going to work this time and there is no need to modify existing files, thanks Daniel for the code review.